mlflow/utils/search_utils.py

```diff
@@ -576,6 +584,48 @@ def paginate(cls, runs, page_token, max_results):
     VALID_SEARCH_KEYS_FOR_MODEL_VERSIONS = set(["name", "run_id", "source_path"])
     VALID_SEARCH_KEYS_FOR_REGISTERED_MODELS = set(["name"])
 
+    @classmethod
+    def _is_valid_identifier_list(cls, value_token):
```

Comment:
maybe `_check_valid_identifier_list` since it doesn't need to return anything (it just throws)